### ISSUE TITLE
Screen stuck on "Loading Notes"

### ISSUE THREAD

**Issue by Gpwaran** [turtl/tracker#422]
Hi,

Today I tried adding a note. I clicked save and it didn't show up. I logged out and back in and it's stuck on "Loading notes" .

I've tried different devices. Tried backing up and restoring onto a new account and nothing has worked.

When I do "restore" the backup it does seem to count the number of notes that were saved and says restored successfully but still stuck on "Loading notes"

Has anyone come across this issue?

**Comment by merechance** [comment]
I also started to experience this issue after several years of using Turtl. I have found out that deleting a few of the last added notes helps. Often it was possible to narrow the issue down to a single note after several attempts of adding them back and restarting the application. Had no time to investigate further what was special about those notes but my notes are usually bookmarked URLs from the browser extension. Currently gave up on the "loading notes" screen and just using individual boards and not All notes. Luckily all my notes are assigned to a board, but if yours are not, I guess you could still try deleting a note in an export file and restoring.

**Comment by LUNTK43** [comment]
Hi there, same issue here. On one specific board, the "Loading notes" screen just freezes.

How could we delete some of the recent files though, if the "All notes" board is not accessible?

Thanks for any help!

**Comment by orthecreedence** [comment]
This is an issue with SQLite ("too many variables"). It's fixed for OSx (https://github.com/turtl/desktop/releases/tag/v0.7.2.6-sqlite-fix) but I'm a dumb dumb and never released the fix for any other platforms. What OS is everyone on? I'll try to do a release. Granted, all this code is a bit dusty now so it might be a bit of acrobatics to get it all going again. We'll see.

**Comment by merechance** [comment]
Thank you, I run both Ubuntu and Windows 10

**Comment by orthecreedence** [comment]
[turtl_core.so.tar.gz](https://github.com/user-attachments/files/16921313/turtl_core.so.tar.gz)

^ Here's a `turtl_core.so` replacement file that should fix the error on linux platforms (with sha256 sum+sig for verification). You'll want to find your local Turtl installation directory then replace `resources/app/build/turtl_core.so` with the bundled one. Please give that a try and let me know if it works.

I'm working on setting up a windows build box to get you a DLL which would allow the same fix on windows, might take a bit longer because development in windows sucks.

**Comment by jomuel** [comment]
@orthecreedence I tried the `turtl_core.so` replacement on an Ubuntu system, and it does nothing. The key you used for the signature is expired, too.

**Comment by orthecreedence** [comment]
When you say "does nothing" the app still loads normally and gets stuck in the same place?

**Comment by jomuel** [comment]
Yes, no change in behavior. It gets stuck on loading the notes, if the number of notes is too large (1000 seems to be the limit)

**Comment by orthecreedence** [comment]
Hey, I did an official new release for Turtl linux desktop (x64): https://github.com/turtl/desktop/releases/tag/v0.7.3

Can you give that a try and let me know if it works?

**Comment by KamMedia** [comment]
@orthecreedence Could you please fix it on Windows ? 
Because sometime I just added a new bookmark to the space that close to the limit and get the Loading notes issue.
If this happened, I have to export all of the notes, manually edit the Json file and then import it back to solve this problem.
I hope this problem could be fix on Windows too, Thanks !
